portal/server/routes/login/index.js

```diff
@@ -60,9 +62,12 @@ router.post('/login', async (req, res) => {
   try {
     const tokenResponse = await api.login(email, password);
 
-    const { token, loginStatus } = tokenResponse;
+    const { token, loginStatus, user: { email: userEmail } } = tokenResponse;
     req.session.userToken = token;
     req.session.loginStatus = loginStatus;
+    req.session.numberOfSendSignInLinkAttemptsRemaining = 2;
+    // TODO DTFS2-6770: Are we right to avoid setting `user.email` (in case this triggers UI elements)?
```

Comment:
Interesting, I didn't set this originally as it wasnt needed. I'm assuming this is due to the requirement to resend email.

As you mentioned - this does draw the top bar if set as `user.email`. This solution is good, i think, but I think it might be worth commenting why we are doing this as `userEmail`. What do you think?